boot/cmdline.go

```diff
@@ -58,10 +61,10 @@ func ModeAndRecoverySystemFromKernelCommandLine() (mode, sysLabel string, err er
 	var modeOk bool
 	mode, modeOk = m["snapd_recovery_mode"]
 
-	// no mode specified gets interpreted as install
+	// no mode specified gets interpreted as ModeCloudImg
```

Comment:
we eventually want no mode specified to actually be an error in the initramfs for UC20, is it possible to instead make this `snapd_recovery_mode=run-cloudimg-rootfs-draft-1` specified on the kernel command line ?

Reply:
At the moment, I'm struggling to make that work. I am hoping to be able to get there though.

Reply:
Would it be possible to also allow it as a cmdline option to snap-bootstrap? As in `--default-mode="blah"` then classic kernelefi initrd will call it with `--default-mode=cloudimg-rootfs`, whereas uc20 will call snap-bootstrap with `--default-mode=install`. I guess we might have to do that, if it proves impossible to add cmdline args reliably in classic case.

Reply:
yes, that sounds a reasonable approach if needed and if the initrd for now need to diverge anyway

Reply:
an option like that seems reasonable :+1:

Reply:
we have worked out a way to have a built-in kernel cmdline in the kernel.efi. if this will be needed in the future a new pr will be opened.